.travis/macos/test

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# man 1 bash
+set -o errexit # set -e
+set -o nounset # set -u
+set -o pipefail
+set -x
+
+sw_vers
+command -v singularity
+singularity version
+singularity help
```

Comment:
I would add a small test for the supported functionalities on macOS, like doing a pull, and a basic `if -f` (check for file existence bc we can not run it)

Reply:
Done as per discussion. Did a search / pull with file check. More tests will be built into the platform at a later date.